tasks/main.yml

```diff
@@ -4,20 +4,20 @@
     name: sudo
     update_cache: yes
     state: present
-  when: "{{ ansible_os_family == 'RedHat' }}"
+  when: ansible_os_family == 'RedHat'
```

Comment:
See the porting guide and look for bare variables: https://docs.ansible.com/ansible/porting_guide_2.0.html